packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/PageContent.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {
+  Field,
+  RichText as JssRichText,
+  useSitecoreContext,
+  LayoutServiceContext,
+  RouteData
+} from '@sitecore-jss/sitecore-jss-nextjs';
+
+export type SitecoreContextValue = LayoutServiceContext & {
+  itemId?: string;
+  route: RouteData;
+};
+
+interface Fields {
+  Content: Field<string>;
+}
+
+type PageContentProps = {
+  params: { [key: string]: string };
+  fields: Fields;
+};
+
+const ComponentContent = (props: any) => {
+  return (
+    <div className={`component content ${props.styles?.replace(/\|/g, ' ')}`}>
+      <div className="component-content">
+        <div className="field-content">
+          {props.children}
+        </div>
+      </div>
+    </div>  
+  )
+}
+
+export const Default = (props: PageContentProps): JSX.Element => { 
+  const { sitecoreContext } = useSitecoreContext<SitecoreContextValue>();
```

Comment:
Don't need to case type for `SitecoreContext` it will not work, see [definition](https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-react/src/enhancers/withSitecoreContext.tsx#L70)